polaris.shopify.com/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import navJSON from '../../../.cache/nav.json';
+import {NavJSON} from '../../types';
+import get from 'lodash.get';
+
+import styles from './Breadcrumbs.module.scss';
+import {uppercaseFirst} from '../../utils/various';
+
+const nav = navJSON as NavJSON;
+
+interface Props {}
+
+function Breadcrumbs({}: Props) {
```

Comment:
I guess we can remove this if there are no Props, no?

```suggestion
function Breadcrumbs() {
```